### ISSUE TITLE
Cannot reproduce TCGA results?

### ISSUE THREAD

**Issue by linusbao** [vkola-lab/tmi2022#26]
I'm having trouble reproducing the results on TCGA. My average performance on TCGA ($\pm$ std across the 5 CV folds) is:

- Accuracy: $0.73244\pm0.009566$
- AUC (one-vs-rest): $0.888692\pm0.006117$
- AUC (one-vs-one): $0.890367\pm0.006113$

Whereas the paper reports:

- Accuracy (%): $82.3\pm1.0$
- AUC (%): $92.8\pm0.3$

# Setup

### Preprocessing
I'm using the provided feature extractor in `/feature_extractor/model.pth` to produce the patch embeddings.

### Hyperparameters
I am training the Graph Transformer with the HPs reported in the paper, as opposed to the HPs that are hardcoded into the repo. As far as I can tell, there were two things that I needed to change in the repo in order to replicate the exact HPs described in the paper:

- Edit `option.py` to change the number of epochs from `120` to `150`
- Edit `main.py` to change the LR scheduler milestones from `[20,100]` to `[30,100]`

(I also tried running an experiment with the default repo HPs as opposed to those described in the paper, and my TCGA results were still inconsistent with the paper's results)

### Data Splits
I split CPTAC into 5 folds, stratifying on the target label to keep the classes balanced across folds. I then trained 5 GTP models (`\scripts\train.sh`), with each GTP model seeing 4/5th of the CPTAC data as the training dataset (`--train_set`) and the remaining 1/5th of CPTAC as the validation set (`--val_set`). Then, I ran evaluation (`\scripts\test.sh`) on each of the 5 models using the full TCGA dataset as the test dataset. The results that I have reported above is the average test set performance of my 5 GTP models.

# Issue
There is a large gap in the accuracy I am achieving on TCGA (73.2%) and the accuracy reported in the paper (82.3%). What could be explaining this discrepancy? 

- Is the provided feature extractor the same exact one used in the paper?
- Are the default HPs in the repo (besides the ones I changed) faithful to the setup used in the paper?
- Is my cross-validation procedure correct?
